docs/design/autoware-interfaces/ad-api/list/api/operation/mode/index.md

```diff
@@ -0,0 +1,44 @@
+# Operation Mode
+
+- {{ link_ad_api('/api/operation/mode/state') }}
```

Comment:
What else will there be added under `operation` in addition to `operation/mode`?
Is it not good if we use `operation_mode` instead?

Reply:
For example, I am considering `/api/operation/command/remote` for remote operation mode.

Reply:
This looks good.
* `/api/operaton_mode/state`
* `/api/external_control/remote(local)/command`

Reply:
Ahange api name https://github.com/autowarefoundation/autoware-documentation/pull/161/commits/b442daaf08bed3155a6266decec08ac0ad7ade2a.